docs/dev/table/connectors/formats/index.zh.md

```diff
@@ -58,10 +58,12 @@ Flink 支持以下格式：
         <tr>
          <td><a href="{% link dev/table/connectors/formats/debezium.zh.md %}">Debezium CDC</a></td>
          <td><a href="{% link dev/table/connectors/kafka.zh.md %}">Apache Kafka</a></td>
+         <td><a href="{% link dev/table/connectors/filesystem.zh.md %}">Filesystem</a></td>
```

Comment:
It's not a new `<td>` column, but just add Filesystem link in the column.

Reply:
Oh, sorry, get it.